### ISSUE TITLE
missing unit supply count, unhappiness breakdown 

### ISSUE THREAD

**Issue by lorduc** [LoneGazebo/Community-Patch-DLL#3255]
_1. Mod version (i.e Date - 4/23):_
Current ( hot fix)

_2. Mod list (if using Vox Populi only, leave blank):_

 Vox Populi, no EUI
_3. Error description:_ 

tool tip doesn't show the source of unhappiness. It shows the current number of unhappiness but it doesnt say where it comes from.
It pretty much says:
current unhappiness 8 
0 from cities
0 from population
but if I build buildings that reduce crime for example, the unhappiness goes down. so i guess its just a problem with the tool tip not displaying it.
im also missing the unit supply count! its just not there where it always used to be!
_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
Sounds like a mod conflict. Enable logging and try again, post your lua log.

G

**Comment by lorduc** [comment]
it was indeed a mod conflict.
I suspected that it was Wonder Race v4, so I disabled that one and it worked. It was either that or because I didnt have (6c) 43 Civs CP enabled.
couple of questions though:

1: How many happiness is tea and crab and etc suppose to give ? Because when I was having this issue I hovered over tea and crab icons and it said:

4 happiness when improved
1 production when worked by a city

but now that the issue is solved it just says:

happiness when improved, (doesnt show a number like it did before, also doesnt show the happiness icon like it did before)
1 production when worked by a city 


so in short, they gave 4 happiness before, now they only give 1... is it supposed to do that ?

Question 2:
Japan's new UA, Is it intended that the bonus culture and science yields start from the second level up onward ? if so do these bonuses scale with era as well, or just level ?

**Comment by bigcat88** [comment]
Just a comment: Wonder Race (v 6) with eui - works perfect. Maybe u will update ur Wonder Race )

And in Vox Populi every luxury give equal amount of happiness depended of ur population(tooltip when u put ur mouse on happiness icon). And yeah, at start of game 1 happiness is normal as I remember(don't remember how much pop u need to get 2, but tooltip always says how much).

On second question I don't know answer.

**Comment by lorduc** [comment]
@bigcat88 thank you :)

**Comment by LoneGazebo** [comment]
#2 yes and yes.
